easybuild/easyblocks/c/cmake.py

```diff
@@ -99,6 +100,10 @@ def configure_step(self):
         add_cmake_opts = {}
         if self.cfg['use_openssl']:
             add_cmake_opts['CMAKE_USE_OPENSSL'] = 'ON'
+        # Use CMP0094 policy for python resolution. This is only available for CMake versions newer than 3.15 included
+        cmake_version = get_software_version('CMake')
+        if LooseVersion(cmake_version) >= LooseVersion('3.15'):
+            add_cmake_opts['MAKE_POLICY_DEFAULT_CMP0094'] = 'NEW'
```

Comment:
```suggestion
            add_cmake_opts['CMAKE_POLICY_DEFAULT_CMP0094'] = 'NEW'
```

https://cmake.org/cmake/help/latest/variable/CMAKE_POLICY_DEFAULT_CMPNNNN.html